docs/data-cookbooks/hello-world.md

```diff
@@ -12,50 +12,52 @@ Example task meant to be a sanity check/introduction to the Cumulus workflows.
 
 ### Workflow Configuration
 
-The [workflow definition](workflows/README.md) can be found in [`cumulus/example/workflows/helloworld.yml`](https://github.com/nasa/cumulus/blob/master/example/workflows/helloworld.yml) under `HelloWorldWorkflow:`
-
-```yaml
-HelloWorldWorkflow:
-  Comment: 'Returns Hello World'
-  StartAt: HelloWorld
-  States:
-    HelloWorld:
-      Parameters:
-        cma:
-          event.$: '$'
-          task_config:
-            buckets: '{$.meta.buckets}'
-            provider: '{$.meta.provider}'
-            collection: '{$.meta.collection}'
-      Type: Task
-      Resource: ${HelloWorldLambdaFunction.Arn}
-      Retry:
-        - ErrorEquals:
-          - Lambda.ServiceException
-          - Lambda.AWSLambdaException
-          - Lambda.SdkClientException
-          IntervalSeconds: 2
-          MaxAttempts: 6
-          BackoffRate: 2
-      Next: WorkflowSucceeded
-    WorkflowSucceeded:
-      Type: Succeed
+A workflow definition can be found in the [template repository hello_world_workflow module](https://github.com/nasa/cumulus-template-deploy/blob/master/cumulus-tf/hello_world_workflow.tf).
+
+```json
+{
+  "Comment": "Returns Hello World",
+  "StartAt": "HelloWorld",
+  "States": {
+    "HelloWorld": {
+      "Parameters": {
+        "cma": {
+          "event.$": "$",
+          "task_config": {
+            "buckets": "{$.meta.buckets}",
+            "provider": "{$.meta.provider}",
+            "collection": "{$.meta.collection}"
+          }
+        }
+      },
+      "Type": "Task",
+      "Resource": "${module.cumulus.hello_world_task_lambda_function_arn}",
+      "Retry": [
+        {
+          "ErrorEquals": [
+            "Lambda.ServiceException",
+            "Lambda.AWSLambdaException",
+            "Lambda.SdkClientException"
+          ],
+          "IntervalSeconds": 2,
+          "MaxAttempts": 6,
+          "BackoffRate": 2
+        }
+      ],
+      "End": true
+    }
+  }
+}
 ```
 
+
 Workflow **error-handling** can be configured as discussed in the [Error-Handling](error-handling.md) cookbook.
 
 ### Task Configuration
 
-The HelloWorld [task itself](workflows/developing-workflow-tasks.md) is defined in [`cumulus/example/lambdas.yml`](https://github.com/nasa/cumulus/blob/master/example/lambdas.yml) under `HelloWorld:`
+The HelloWorld [task](workflows/developing-workflow-tasks.md) is provided for you as part of the `cumulus` terraform module, no configuration is needed.
```

Comment:
Are you intending to link to the "developing tasks" doc or the hello world task definition?

Reply:
That was the *original* intent of it.  I just got rid of the redundant 'itself' here.      

I don't know that the task definition is particularly more or less helpful, but I can remove the link.

Reply:
34b46e815